src/woo_publications/conf/base.py

```diff
@@ -135,13 +135,14 @@
 #
 
 REST_FRAMEWORK = BASE_REST_FRAMEWORK.copy()
-REST_FRAMEWORK["PAGE_SIZE"] = 100
+REST_FRAMEWORK["PAGE_SIZE"] = 10
+REST_FRAMEWORK["MAX_PAGE_SIZE"] = 100
```

Comment:
this is not a DRF setting and adding custom settings to DRF like this is *very* confusing.

For this particular case, you can just define the hardcoded number in `DynamicPageSizePagination` instead of reading it from some sort of setting.